src/cascadia/TerminalApp/CascadiaSettingsSerialization.cpp

```diff
@@ -18,12 +18,12 @@ using namespace winrt::Windows::Storage;
 using namespace winrt::Windows::Storage::Streams;
 using namespace ::Microsoft::Console;
 
-static const std::wstring FILENAME { L"profiles.json" };
-static const std::wstring SETTINGS_FOLDER_NAME{ L"\\Microsoft\\Windows Terminal\\" };
+static const std::wstring_view FILENAME { L"profiles.json" };

```

Comment:
I believe that these all can be made `constexpr` now as well.

Reply:
good point, updating the commit

Reply:
done